### DISCUSSION TITLE
White Screen Issue on Production for Data Catalog App

### DISCUSSION THREAD

**Discussion by marlubo** [GetDKAN/dkan#4103] [Q&A]
Hello community,

I hope you are doing well. I am currently facing a perplexing issue with my production environment that I would like to seek assistance for.

Problem Description:
On my development environment using DDEV, everything works fine. However, when I deploy my application to the production server, I encounter a white page accompanied by the error message: "Unexpected Application Error! 404 Not Found." Upon inspecting the browser console, I see a series of 404 errors related to resource loading, such as "Rubik-Regular.woff."

Details:
The physical location of the build output is abc.com/public_html/docroot/frontend/build/static.
The expected location for the frontend is abc.com/public_html/docroot/.
Attempts to Resolve:

I have updated the jsonpackage and viteconfig.
I have verified the configurations in both the development and production environments to ensure consistency.
Despite these efforts, the issue persists, and the 404 errors persist.
Request for Assistance:
I am reaching out to the community for guidance and suggestions on how to troubleshoot and resolve this issue. If anyone has encountered a similar problem or has insights into potential solutions, I would greatly appreciate your assistance.

Thank you in advance for your time and expertise.

Best regards,
Colmarer
![frontend](https://github.com/GetDKAN/dkan/assets/75080007/967c5a96-ffa7-4bbd-b1c2-0e78521d52d8)
![viteconfig](https://github.com/GetDKAN/dkan/assets/75080007/b4b94a34-6cb6-4fa1-89c3-d9e34daa977d)
![packagejson](https://github.com/GetDKAN/dkan/assets/75080007/801cd9e1-df3e-42bb-b0e7-317681acb52f)

**Comment by brdunfield** [comment]
@colrmer It looks like you've put the `base` property in your vite config within the build options, when it should be raised up to the same level as `build` and `plugins`.

Also, by default the catalog app should not be viewable at the `/docroot` path, unless this is something you have configured elsewhere. If fixing the vite config doesn't help, could you share any drupal or ddev configuration you have done for the frontend so we can investigate further?
